pid/__init__.py

```diff
@@ -162,7 +158,7 @@ def create(self):
         self.fh = open(self.filename, 'a+')
         if self.lock_pidfile:
             try:
-                fcntl.flock(self.fh.fileno(), fcntl.LOCK_EX | fcntl.LOCK_NB)
+                fasteners.lock.try_lock(self.fh)
```

Comment:
https://github.com/harlowja/fasteners/blob/master/fasteners/lock.py#L257

`try_lock` expects a python lock not a file handler ?